docs/sources/operators-guide/monitor-grafana-mimir/dashboards/alertmanager-resources/index.md

```diff
@@ -1,10 +1,11 @@
 ---
-title: "Grafana Mimir Alertmanager resources dashboard"
-menuTitle: "Alertmanager resources"
-description: "View an example Alertmanager resources dashboard."
-weight: 20
 aliases:
-  - ../../visualizing-metrics/dashboards/alertmanager-resources/
+  - /../../visualizing-metrics/dashboards/alertmanager-resources/
```

Comment:
This edit looks wrong to me. It was relative, now it's absolute. I just manually tried, and doesn't work anymore.

_Same applies to all other places._

I open a PR to fix them.

Reply:
Please build the entire website for testing.